pages/doc.js

```diff
@@ -141,6 +137,8 @@ export default class Documentation extends Component {
           }
         )
       })
+    }).catch(()=>{
```

Comment:
will it catch any error? can it cause an infinite loop? can we actually detect a specific error to be safe here?

Reply:
This will catch any http request error.
This cannot cause an endless loop.
Yes, here we can see the specific error of the http request: the code and text of the error.

Reply:
👍 My bad. Didn't recognize this is a promise catch.